frontend/src/main/web/src/components/TableCell.js

```diff
@@ -0,0 +1,86 @@
+import React, { PropTypes } from 'react'
+import { merge } from 'lodash'
+import { View } from './'
+
+const classes = {
+  base: {
+    fld: '',
+    ai: 'Ai(c)',
+    h: 'H(r2)',
+    p: 'Py(rq) Px(rh)'
+  },
+  tight: {
+    p: 'Py(rq) Px(rq)'
+  },
+  hideSmall: {
+    d: 'D(f) D(n)--lesm'
+  },
+  '1': {
+    w: 'W(1/6)'
+  },
+  '2': {
+    w: 'W(2/8) W(2/6)--md W(1/6)--lg'
+  },
+  '3': {
+    w: 'W(3/8) W(2/6)--md'
+  }
+}
+
+/**
+ * A Table Cell for use in Table Rows
```

Comment:
Maybe "... for use in TableRow components" so it is clear exactly what it means (I assume that is what it is talking about).